hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/AttributeValidatorImpl.java

```diff
@@ -91,7 +91,7 @@ public void validateExpiry(long expiry) {
                 context.configuration().getConfigData(EntitiesConfig.class).maxLifetime();
         final var now = context.consensusNow().getEpochSecond();
         final var expiryGivenMaxLifetime = now + maxEntityLifetime;
-        validateTrue(expiry > now && expiry <= expiryGivenMaxLifetime, INVALID_EXPIRATION_TIME);
+        validateTrue(expiry > now && expiry < expiryGivenMaxLifetime, INVALID_EXPIRATION_TIME);
```

Comment:
I think Iris had a PR that changed this one but it was rejected. Does anyone knows if this change is correct? As far as I can see the old code works the same way:
`return then.isAfter(consensusNow) && then.isBefore(expiryGivenMaxLifetime);`

Reply:
@tinker-michaelj wrote the original code in `StandardizedAttributeValidator`. We should check with him.

Reply:
I agree with @vtronkov, the [corresponding check](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/validation/ContextOptionValidator.java#L145) in mono-service is a strict inequality, `<` not `<=`.

Reply:
Some tests were failing so I made that change to pass those tests. If you change this back, need to double check those tests.